Storybook builds are failing without failing CircleCI builds

Two problems:
1. Storybook builds are failing with an error
1. This error isn't recognized by CircleCI as a failure of the build stage

The first should probably be fixed along with the second, so that we get proper awareness of Storybook build failures.

An example: https://circleci.com/gh/mozilla/testpilot/9687

Relevant error output under "Storybook Deploy":
```
#!/bin/bash -eo pipefail
./bin/deploy-storybook.sh

info @storybook/react v3.4.0-alpha.1
info 
info => Loading custom .babelrc
info => Loading custom addons config.
info => Loading custom webpack config (full-control mode).
info Building storybook ...
ERR! Failed to build the storybook
ERR! Module not found: Error: Can't resolve '../frontend/build/static/styles/main.css' in '/home/circleci/project/.storybook-9fac21bd13e7a83796d58d885d559bc6f8ceb18f'
```

--- a/.storybook/config.js
+++ b/.storybook/config.js
@@ -2,8 +2,7 @@
 
 import { configure } from '@storybook/react';
 
-import '../frontend/build/static/styles/experiments.css';
-import '../frontend/build/static/styles/main.css';
+import '../frontend/build/static/app/app.css';
 
 const reqInSrcTree = require.context('../frontend/src/app', true, /\/stories.jsx?$/);
 

--- a/bin/deploy-storybook.sh
+++ b/bin/deploy-storybook.sh
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -ex
 
 STORYBOOK_BUCKET=${STORYBOOK_BUCKET:-testpilot-storybook.dev.mozaws.net}
 if [ -z "$STORYBOOK_BUCKET" ]; then

